tron/trondaemon.py

```diff
@@ -155,7 +165,13 @@ def _build_context(self, options):
             working_directory=options.working_dir,
             umask=0o022,
             pidfile=pidfile,
-            signal_map={},
+            signal_map={
```

Comment:
@qui, we do only need to reset the `SIGINT` handler, but its actually not true that setting a default handler is different. The reason is that by default, all signals have handlers, namely `SIG_DFL` (except `SIGINT`, which has `default_int_handler` instead). You can actually check this by running `signal.getsignal` for any signal you want in a Python interpreter. 

I decided to set all of them to default for brevity, because in the event anyone else attempts to set things (which shouldn't happen but you never know), it shouldn't mess with us weirdly.